android/src/main/java/com/reactnative/googlefit/CalorieHistory.java

```diff
@@ -164,6 +164,16 @@ private void processDataSet(DataSet dataSet, WritableArray map, boolean basalCal
                     }
                 }
                 stepMap.putDouble("calorie", dp.getValue(field).asFloat() - basal);
+
+                /** Checks if data point was added manually by user */
+                DataSource ds = dp.getOriginalDataSource();
+                String streamId = ds.getStreamIdentifier();
+                if (streamId.toLowerCase().indexOf("user_input") != -1) {
```

Comment:
can you explain what this line is doing a little, basically why `indexOf("user_input")`

Reply:
You want me to add a comment line with the explanation in the code ?

Reply:
@sash20m  nope, just generally wonder why it checks the specific string `user_input`, does it come from any Google Fit convention or something?